### DISCUSSION TITLE
I download cx_freeze on python 3.9 in any way. There is no cxfreeze postinstall file in the/Python/Scripts/   path

### DISCUSSION THREAD

**Discussion by YANCC139** [marcelotduarte/cx_Freeze#1739] [Q&A]
I download cx_freeze on python 3.9 in any way. There is no cxfreeze postinstall file in the/Python/Scripts/   path,there is only cxfreeze-quickstart.exe and cxfreeze.exe

**Comment by marcelotduarte** [comment]
If I understand what you are saying, this is correct behavior.
If you install cx_Freeze with:
`pip install --upgrade cx_Freeze`
Then, you can check if it is installed:
cxfreeze --version
